app/src/main/res/values/strings.xml

```diff
@@ -7,46 +7,46 @@
     <string name="dialog_error_title_podcast_invalid_feed">Invalid Podcast Feed</string>
     <string name="dialog_generic_button_cancel">Cancel</string>
-    <string name="dialog_generic_button_okay">Okay</string>
+    <string name="dialog_generic_button_okay">OK</string>
     <string name="dialog_generic_details_button">Show details</string>
     <string name="dialog_generic_details_default">No details available</string>
     <string name="dialog_metered_download_episode_button_okay">Download anyway</string>
-    <string name="dialog_metered_download_episode_message">Your device is not connected to a Wifi network. Do you want to download this episode anyway?</string>
-    <string name="dialog_metered_download_episode_title">Download without Wifi</string>
+    <string name="dialog_metered_download_episode_message">Your device is not connected to Wi-Fi. Download episode anyway?</string>
+    <string name="dialog_metered_download_episode_title">Download without Wi-Fi</string>
     <string name="dialog_metered_network_button_okay_default">Proceed anyway</string>
     <string name="dialog_metered_network_message_default">Your device is connected to a metered network.</string>
     <string name="dialog_metered_network_title_default">Metered Network</string>
     <string name="dialog_metered_update_collection_button_okay">Update anyway</string>
-    <string name="dialog_metered_update_collection_message">Your device is not connected to a Wifi network. Do you want to update your podcasts anyway?</string>
-    <string name="dialog_metered_update_collection_title">Update without Wifi</string>
+    <string name="dialog_metered_update_collection_message">Your device is not connected to Wi-Fi. Update your podcasts anyway?</string>
+    <string name="dialog_metered_update_collection_title">Update without Wi-Fi</string>
     <string name="dialog_opml_import_button_okay">Import</string>
     <string name="dialog_opml_import_details_button">Show details</string>
     <string name="dialog_opml_import_details_default">No details available</string>
-    <string name="dialog_opml_import_message_error">Unable to import OPML file. No podcasts found.</string>
-    <string name="dialog_opml_import_message">Do you want to add new podcasts? Duplicates will be removed before importing. Number of podcasts found:</string>
+    <string name="dialog_opml_import_message_error">Could not import OPML file. No podcasts found.</string>
+    <string name="dialog_opml_import_message">Add new podcasts? Duplicates will be removed before importing. Number of podcasts found:</string>
     <string name="dialog_opml_import_title">OMPL Import</string>
     <string name="dialog_show_notes_close">Close</string>
-    <string name="dialog_show_notes_title">Show Notes</string>
-    <string name="dialog_yes_no_message_add_up_next">Do you want to start listening to this episode right away?</string>
+    <string name="dialog_show_notes_title">Shownotes</string>
```

Comment:
Not 100% sure about this. One more instance in the descriptions.

Reply:
I think the "Show Notes" is the correct English spelling for this. I am not a native speaker, but a lot of podcasts spell it that way.

But I understand the potential for confusion. I could rename the XML string from `name="dialog_show_notes_title"` to something less confusing. Any suggestion?

Reply:
@y20k What do you think of my solution to call it "episode show notes"?

Reply:
yeah. that sounds like a good solution.